### PR TITLE
Create addon.txt

### PR DESCRIPTION
**Pull request by rawatakshay**
hi can you merge my code with your main branch

### CODE REVIEW

File: addon.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+hey adding few lines
```
Comment (AshishGusain17) [inline addon.txt:1]: ok...checking your lines

Comment (AshishGusain17) [inline addon.txt:1]: ok   what are those lines

File: addon.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+this branch is created by rawat akshay
```
Comment (AshishGusain17) [inline addon.txt:3]: nice akshay

File: addon.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+this branch is created from main
```
Comment (AshishGusain17) [inline addon.txt:5]: oh nice

File: addon.txt (hunk 1 above)
Comment (rawatakshay) [reply addon.txt:1]: just to tell we are addign few lines

File: addon.txt (hunk 2 above)
Comment (AshishGusain17) [inline addon.txt:3]: just checking on this again

**Review by AshishGusain17 — CHANGES_REQUESTED** [review]
please add 1 more line

**Comment by rawatakshay** [comment]
added 4th line...review again

**Pull request merged**